examples/axes_grid1/make_room_for_ylabel_using_axesgrid.py

```diff
@@ -7,61 +7,65 @@
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 from mpl_toolkits.axes_grid1.axes_divider import make_axes_area_auto_adjustable
 
+import matplotlib.pyplot as plt
 
-if __name__ == "__main__":
+def ex1():
+    plt.figure(1)
```

Comment:
```suggestion
    plt.figure()
```
We don't make any use of explicit figure numbers, so let's remove them to reduce clutter. Also in the other examples.